### DISCUSSION TITLE
Can we set height of textarea?

### DISCUSSION THREAD

**Discussion by Duckinm** [payloadcms/payload#261] [Q&A] [answered]
As the Topic said.

**Comment by Duckinm** [comment]
One more, Did "textarea" are being broke right now? After mouse drag is ended, I can't drag it again.

![Screen Shot 2564-08-02 at 3 45 17 AM](https://user-images.githubusercontent.com/29366271/127784851-2429af0a-9aef-498a-b4e0-239f7373e3c2.png)

**Comment by jmikrut** [comment, marked as answer]
Hey @Duckinm —

You can set the height of a `textarea` like this:

```js
const textareaField = {
  name: 'yourFieldHere',
  type: 'textarea',
  admin: {
    rows: 10, // set however many rows you want the textarea to appear with
  }
}
```

Also, from your screenshot, it looks like when you resized your textarea, you dragged it _**behind**_ the field to the right, which is no longer clickable / resizable because that field is sitting on top of your actual clickable textarea handle. Maybe a good future option would be for us to add a field property to textareas that disables resizing.

What do you think?

**Comment by Duckinm** [comment]
Great! The answer really helpful and the resize:none would be also nice. Keep it up good works!
